active/Win32/Window_and_Application_API_Spec.md

```diff
@@ -0,0 +1,655 @@
+     {
+        string[] arguments = e.Arguments.Split(' ');
+	    // Assuming MainPage has a public property called FileName 
+        page.FileName = arguments[0];
+     }	
+
+    Window.Current.Content = page;
+    Window.Current.Activate();
+    window.Activate();
+}
+```
+
+
+## Application.RequiresPointerMode property
+_Behavior update to existing API when running as Desktop app_
+
+Gets or sets whether a UWP app supports mouse mode, which emulates pointer interaction experiences 
+with non-pointer input devices such as an Xbox gamepad or remote control. (All nested elements inherit 
+this behavior.)
+
+> Note: this property is ignored in a Desktop app
+
+## Application.OnWindowCreated(XamlWindowCreatedEventArgs) method
+_Behavior update to existing API when running as Desktop app_
+
+Invoked when the application creates a window.
+
+```CS
+protected virtual void OnWindowCreated(XamlWindowCreatedEventArgs args)
+```
+
+### Remarks
+- In a UWP app, this method is called once for the creation of the main window, once for each hosted window, 
+and once each time CreateNewView is called. 
+- In a Desktop app, this method is called every time a Window object is created.
+
+## XamlWindowActivatedEventArgs class
+
+Contains the windows activation state information returned by the Window.Activated event.
+
+> Issue: should rename these to avoid collision.
+
+> Spec note: this is a new XAML version of the existing 
+Windows.UI.Core.[WindowActivatedEventArgs](https://docs.microsoft.com/uwp/api/Windows.UI.Core.WindowActivatedEventArgs).
+
+## XamlWindowActivatedEventArgs properties
+
+**Handled**
+
+Gets or sets whether the window activation changed event was handled.
+ **true** if the event has been handled by the appropriate delegate; **false** if it has not.
+
+```CS
+public bool Handled { get; set; }
+```
+
+**WindowActivationState**
+
+Gets the activation state of the window at 
+the time the event was raised.
+
+```CS
+public CoreWindowActivationState WindowActivationState { get; }
+```
+
+## XamlVisibilityChangedEventArgs class
+
+Contains the arguments returned by the event fired when a Window instance's visibility changes.
+
+> Spec note: this is a new Xaml version of the existing 
+Windows.UI.Core.[VisibilityChangedEventArgs](https://docs.microsoft.com/uwp/api/Windows.UI.Core.VisibilityChangedEventArgs).
+
+
+### XamlVisibilityChangedEventArgs
+
+Contains the arguments returned by the event fired when a Window instance's visibility changes.
+
+> Spec note: this is a new XAML version of the existing 
+Windows.UI.Core.[VisibilityChangedEventArgs](https://docs.microsoft.com/uwp/api/Windows.UI.Core.VisibilityChangedEventArgs).
+
+**Handled**
+
+Gets or sets whether the window activation changed event was handled. **true** 
+if the event has been handled by the appropriate delegate; **false** if it has not.
+
+**Visible**
+
+Gets whether the window is visible or not. **true** if the window is visible; otherwise is **false**.
```

Comment:
"is" or "was"? This is an "-ed" event, so the property presumably refers to whether the window was made visible or not, rather than referring to the window's current visibility state (which you can obtain from the Window.Visible property.)